src/content/docs/reference/api-references.md

```diff
@@ -0,0 +1,57 @@
+---
+title: API references
+description: How to format API reference entries.
+---
+Consistent formatting makes it easier for people to find the information they're looking for!
+
+When adding or editing a reference entry, please start your entry with the following standard formatting:
+
+```astro
+##### `page.url.first`
+
+<p>
+
+**Type:** `string | undefined`<br />
+<Since v="4.12.0" />
+</p>
+```
+
+## Things to check
+
+- The entry has a heading so that it can be directly linked to.
+- The heading text is the name of the property/item, formatted as inline code. (e.g. `getStaticPaths()`, `width`)
+- There is a wrapping `<p></p>` that includes an initial blank line, and then the type, default (if applicable), and version number added each on individual lines.
+- The text entry starts with a **definition** of what the property **is** or **does**.
+
```

Comment:
I don't know if this is useful, but maybe we could mention `configuration-reference.mdx` since the generated file uses this format. Something:

> If in doubt while adding an entry in your editor, you can check the syntax used in `configuration-reference.mdx` as a reference.

Otherwise, LGTM! Thanks 🎉